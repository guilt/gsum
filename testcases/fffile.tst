Name: fffile
Description: Hashing a file filled with 0xFF bytes
File: fffile.txt 8B fill ff
AlgoMatrix: default
