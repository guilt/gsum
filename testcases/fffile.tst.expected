Hash-default: 12a3ae445661ce5dee78d0650d33362dec29c4f82af05e7e57fb595bbbacf0ca
